src/patternfly/components/InputGroup/examples/input-group-example.hbs

```diff
@@ -0,0 +1,59 @@
+<div class="pf-c-input-group">
+  {{#> button btnClass="pf-m-secondary"}} Button {{/button}}
+  {{#> form-controls controlType="textarea" name="textarea1" id="textarea1" aria-label="textarea example"}}
+{{/form-controls}}
+    {{#> button btnClass="pf-m-tertiary"}} Button {{/button}}
+</div> 
+<br>
+<br>
+<div class="pf-c-input-group">
+  {{#> form-controls controlType="textarea" name="textarea2" id="textarea2" aria-label="textarea example"}}
+{{/form-controls}}
+    {{#> button btnClass="pf-m-tertiary"}} Button {{/button}}
+</div> 
+<br>
+<br>
+<div class="pf-c-input-group">
+{{#> dropdown id="dropdown-example-collapsed"}}Dropdown{{/dropdown}}
+  {{#> form-controls controlType="input" input="true" type="text" id="textInput3" name="textInput3" placeholder="Standard input example" aria-label="standard input example" }}
```

Comment:
Since `placeholder` text is intended to provide hints about the expected value, then we should avoid using it in our examples for anything but it's intended purpose. One of the biggest usability/accessibility issues related to `placeholder` is that it's misused as a label for the input field, and using it here in this way might help to perpetuate this mis-usage.

Reply:
that makes a lot of sense.